ci/worker_files/speed-tester-worker.sh

```diff
@@ -15,10 +15,10 @@ sleep 15
 log "Manager is live"
 
 log "Running Speed Tester NF"
-cd ~/repository/examples/speed_tester
+cd ~/repository/examples/speed_tester || echo "ERROR: couldn't cd into speed_tester" && return 1
 ./go.sh 1 -d 1 &>~/speed_stats &
 spd_tstr_pid=$?
-if [ $spd_tstr_pid -ne 0 ] 
+if [ $spd_tstr_pid -ne 1 ]
```

Comment:
most of the changes look like simple formatting type issues but this is a logic change -- if this was done automatically can you ensure it is correct?